.jcheck/conf

```diff
@@ -18,12 +18,18 @@ domain=openjdk.org
 files=.*\.cpp|.*\.hpp|.*\.c|.*\.h|.*\.java|.*\.cc|.*\.hh|.*\.m|.*\.mm|.*\.md|.*\.gmk|.*\.m4|.*\.ac|Makefile
 ignore-tabs=.*\.gmk|Makefile
 
+[checks "merge"]
+message=Merge
+
 [checks "reviewers"]
-committers=1
+reviewers=1
 ignore=duke
 
 [checks "committer"]
 role=committer
 
+[checks "issues"]
+pattern=^([124-8][0-9]{6}): (\S.*)$
+
 [checks "problemlists"]
```

Comment:
I'd remove any changes to jcheck by adding it to your gitignore or something.

Reply:
OTOH may be the intention _is_ to change the jcheck for project repo to align it with upstream in which case ignore my first comment in thread.

Reply:
I just didn't want to have to "redo"  the undo of these changes for the next PR. I think if those design docs are still useful, we ought to incorporate them in the JEP or as a comment in `shenandoahGeneration.hpp`?

We originally disabled the jcheck rule that required PR's to have a JBS ticket because we weren't really using JBS at the time, but I think the intention going forward is that we _are_ using JBS, so we should have the rule again.